library/src/main/java/com/novoda/downloadmanager/lib/DownloadService.java

```diff
@@ -88,6 +88,8 @@
     private NetworkChecker networkChecker;
     private DestroyListener destroyListener;
 
+    private final NotificationCreatedListener notificationsCreatedCallback = new NotificationCreatedListener(this);
```

Comment:
is it worth `new'ing` up in the `onCreate` to avoid the possibility of the service being interacted with before it has been fully created?